src/main/java/CI/ContinuousIntegrationServer.java

```diff
@@ -62,8 +62,8 @@ public void handle(String target,
                     handler.request_push();
                     response.getWriter().println("CI job done");
                 }
-                catch (IOException ioe) {
-                    ioe.printStackTrace();
+                catch (Exception ex) {
```

Comment:
"catch (Exception)" is a bad pattern, as it catches all possible exceptions. A better approach would be to catch only JSONException's here.

Even then, the "catch" would cover too much code. It only needs to cover the line where the GitEvent is created.